src/sentry/reprocessing2.py

```diff
@@ -206,11 +212,16 @@ def reprocess_event(project_id: int, event_id: str, start_time: float) -> None:
 
     # Step 1: Fix up the event payload for reprocessing and put it in event
     # cache/event_processing_store
+
+    if consumer_type == ConsumerType.Transactions:
+        cache = transaction_processing_store
+    else:
+        cache = event_processing_store
```

Comment:
Nit: I would avoid the name `cache` as it could be confused with the `cache` services from django. `processing_store` might be a less ambiguous but still generalized name.